i18n/fi.po

```diff
@@ -0,0 +1,1222 @@
+"\n"
+"                    <a href=\"%(integer)s\">arbitrary precision integers</"
+"a>,\n"
+"                    <a href=\"%(rational)s\">rationals</a>,\n"
+"                    <a href=\"%(float)s\">and floats</a>\n"
+"                    "
+msgstr ""
+"\n"
+"                    <a href=\"%(integer)s\">mielivaltaisen tarkat "
+"kokonaisluvut</a>,\n"
+"                    <a href=\"%(rational)s\">rationaaliluvut</a>,\n"
+"                    <a href=\"%(float)s\">ja liukuluvut</a>\n"
+"                    "
+
+#: templates/features.html:58
+msgid "Noncommutative symbols"
+msgstr "Epävaihdannaiset"
+
+#: templates/features.html:59
+msgid "Pattern matching"
+msgstr "Kuvion täsmäys"
+
+#: templates/features.html:65
+msgid "Polynomials"
+msgstr "Polynomit"
+
+#: templates/features.html:69
+msgid " division, gcd, ... "
+msgstr "jaollisuus, syt, ..."
+
+#: templates/features.html:70
+msgid "Factorization"
+msgstr "Tekijöihin jakaminen"
+
+#: templates/features.html:71
+msgid "Square-free decomposition"
+msgstr "Neliötön hajotelma"
+
+#: templates/features.html:72
+msgid "Gröbner bases"
+msgstr "Gröbnerin kannat"
+
+#: templates/features.html:73
+msgid "Partial fraction decomposition"
+msgstr "Osamurtohajotelma"
+
+#: templates/features.html:74
+msgid "Resultants"
+msgstr "Resultantit"
+
+#: templates/features.html:80
+msgid "Calculus"
+msgstr "Differentiaalilaskenta"
+
+#: templates/features.html:82
+msgid "Integration Documentation"
+msgstr "Integrointidokumentit"
+
+#: templates/features.html:85
+msgid "Limits:"
+msgstr "Raja-arvot"
+
+#: templates/features.html:89
+msgid "Differentiation"
+msgstr "Derivointi:"
+
+#: templates/features.html:92
+msgid "Integration:"
+msgstr "Integrointi:"
+
+#: templates/features.html:93
+msgid "It uses extended Risch-Norman heuristic"
+msgstr "Se käyttää laajennettua Risch-Norman - heuristiikkaa"
+
+#: templates/features.html:95
+msgid "Taylor (Laurent) series"
+msgstr "Taylor (Laurent) - sarjat"
+
+#: templates/features.html:101
+msgid "Solving equations"
+msgstr "Yhtälönratkaisu"
+
+#: templates/features.html:105
+msgid "Polynomial equations"
+msgstr "Polynomiyhtälöt"
+
+#: templates/features.html:106
+msgid "Algebraic equations"
+msgstr "Algebralliset yhtälöt"
+
+#: templates/features.html:107
+msgid "Differential equations"
+msgstr "Differentiaaliyhtälöt"
+
+#: templates/features.html:108
+msgid "Difference equations"
+msgstr "Differenssiyhtälöt"
+
+#: templates/features.html:109
+msgid "Systems of equations"
+msgstr "Yhtälöryhmät"
+
+#: templates/features.html:110
+msgid "Diophantine equations:"
+msgstr "Diofantoksen yhtälöt"
+
+#: templates/features.html:116
+msgid "Combinatorics"
+msgstr "Kombinatoriikka"
+
+#: templates/features.html:120
+msgid "Permutations"
+msgstr "Permutaatiot"
+
+#: templates/features.html:121
+msgid "Combinations"
+msgstr "Kombinaatiot"
+
+#: templates/features.html:122
+msgid "Partitions"
+msgstr "Ositukset"
+
+#: templates/features.html:123
+msgid "Subsets"
+msgstr "Aliryhmät"
```

Comment:
The code appears to generate subsets of a finite set. Hence we should have "Osajoukot".